gc/base/HeapMap.hpp

```diff
@@ -255,6 +255,18 @@ class MM_HeapMap : public MM_BaseVirtual
 		return false;
 	}
 	
+	MMINLINE uintptr_t*
+	getSlotPtr(size_t index) const
```

Comment:
My mistake, shouldn't have const-qualified this:
```suggestion
	getSlotPtr(size_t index)
```